GameArena.java

```diff
@@ -115,7 +119,7 @@ private void init(int width, int height, boolean createWindow)
 		renderingHints.put(RenderingHints.KEY_TEXT_ANTIALIASING, RenderingHints.VALUE_TEXT_ANTIALIAS_LCD_HRGB);
 
 		try {
-			Class rh = Class.forName("java.awt.RenderingHints");
+			Class<?> rh = Class.forName("java.awt.RenderingHints");
```

Comment:
Which JDK did this raise a warning on?

Reply:
I believe the VS Code extension runs Java 8. However, I'm not certain about that